docs/codeactions/code-actions.md

```diff
@@ -0,0 +1,92 @@
+---
+id: codeactions
+title: Code Actions
+---
+
+
+Code Actions are used to provide automatic refactoring or rewriting of your current code in order to enhance its readability or save users' time.
+
+In Visual Studio Code, to use them you can click on the yellow light bulb appearing on your code, near the cursor and then, to pick the desired action to perform on the code. Alternatively, you can use the `editor.action.quickFix` shortcut.
+
+For other editors this should also be available via specified shortcut.
+
+Here is a list of the code actions supported by Metals. Some of these code actions support much more complex use cases, which you would eventually discover through your code, by clicking on the magic 💡, as it appears, to get your surprise!
+
+## Convert To Named Arguments
+
+This code action adds the name to parameters passed to a method call or a class constructor.
+
+[Convert To Named Arguments Code Action](https://github.com/scalameta/metals/docs/codeactions/gifs/ConvertToNamedArguments.gif)
```

Comment:
```suggestion
![convert to named arguments code action](./gifs/converttonamedarguments.gif)
```

to show the gif in the webpage instead of just a link to gif